apps/judicial-system/web/src/routes/Court/InvestigationRequest/Ruling/StepTwo/RulingStepTwoForm.tsx

```diff
@@ -53,12 +54,16 @@ const RulingStepTwoForm: React.FC<Props> = (props) => {
             Úrskurður og kæra
           </Text>
         </Box>
+        <Box component="section" marginBottom={7}>
```

Comment:
Do you think it would be cleaner if we placed this Box inside the CaseNumbers component and passed in marginBottom as a prop?